README.md

```diff
@@ -19,8 +38,17 @@ corepack yarn
 corepack yarn dev
 ```
 
-This command starts a local development server and opens up a browser window.
-Most changes are reflected live without having to restart the server.
+This command builds **/uppy/docs** and **/uppy.io**, and shows them in a
+browser. Most changes are reflected live without having to restart the server.
+
+### Lint
+
+```sh
+corepack yarn run format
+corepack yarn lint
+```
+
+This command lints **/uppy/docs** and **/uppy.io**.
```

Comment:
```suggestion
N.B.: These commands will also report lint errors from the `docs` folder that's hosted on the `transloadit/uppy` repository.
```

Reply:
My version is more consistent with the phrasing of other command descriptions.